ps.go

```diff
@@ -65,6 +70,7 @@ func psAction(clicontext *cli.Context) error {
 func printContainers(ctx context.Context, clicontext *cli.Context, containers []containerd.Container) error {
 	trunc := !clicontext.Bool("no-trunc")
 	all := clicontext.Bool("all")
+	quiet := clicontext.Bool("quiet")
 
 	w := tabwriter.NewWriter(clicontext.App.Writer, 4, 8, 4, ' ', 0)
 	fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES")
```

Comment:
header needs to be hidden for quiet mode

Reply:
Nice catch :) let me fix it.

Reply:
Fixed.